nolooking/docker-compose.yml

```diff
@@ -5,6 +5,7 @@ services:
     environment:
       APP_HOST: nolooking_server_1
       APP_PORT: 4444
+      PROXY_AUTH_WHITELIST: "/pj"
```

Comment:
Just wanted to check if this is not the equivalent as turning off the authentication (in the App Proxy) all together? If I recall, the whole app was hosted at `/pj` right?

Reply:
Yeah it is, I think this is OK for now (since I believe the only risk of leaked onion is that someone could send you money and open channels). We are separating endpoints out in an upcoming PR
https://github.com/chaincase-app/nolooking/pull/55

Reply:
Ok understood, do you want to wait for that to be ready to avoid pushing another update. Or would you like this merged ASAP?

Reply:
Let's do it now since the app is broken until then, and as long don't another update PR sometime next week. Thanks!